src/main/java/org/openpnp/machine/reference/driver/GcodeDriver.java

```diff
@@ -188,6 +187,33 @@ public void home(ReferenceHead head) throws Exception {
         for (ReferenceDriver driver : subDrivers) {
             driver.home(head);
         }
+
+        /*
+                The head camera for nozzle-1 should now be (if everythings homed correctly)
+                directly above the homing pin in the machine bed, use the head camera scan
+                for this and make sure this is exactly central - otherwise we move the camera
+                until it is, and then reset all the axis back to 0,0,0,0 as this is calibrated
+                home.
+         */
+        Part homePart=Configuration.get().getPart("FIDUCIAL-HOME");
```

Comment:
I'd like to see this part be configurable, using a part dropdown like in the feeder configuration, for instance. If that is overly difficult, let's at least make it a configurable parameter in the XML and I will add the UI for it later. It should just be something like homingFiducialPartId, for instance.